cli/bin/exec.js

```diff
@@ -129,13 +129,25 @@ function execGrainrun(file, options, program, execOpts = { stdio: "inherit" }) {
     preopens[guestDir] = hostDir;
   });
 
+  const rawArgs = process.argv;
+  const endOptsIndex = rawArgs.findIndex((x) => x === "--");
+  const args = rawArgs.slice(endOptsIndex === -1 ? Infinity : endOptsIndex + 1);
```

Comment:
This PR fails on Windows because you need to pre-process the `process.argv` instead of post-process it after commander messes around. I believe @spotandjake saw a way to do that.

Reply:
Yes as shown by the types [here](https://github.com/tj/commander.js/blob/33195f189b1ffa568c232503fb6ac0cf5548eb74/typings/index.d.ts#LL674C6-L674C6), it looks as though we could get process the arguments and then just pass the pre proccessed arguments in.